app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/representee/summary.scala.html

```diff
@@ -46,7 +47,12 @@
 
     @cyaRow(
         messages("representee.cyaLabel.personRepresented"),
-        Html(s"""${answers.name.makeSingleName()}<br>${answers.id}"""),
+        Html(s"""${
+            answers.name.makeSingleName()}<br>
+            ${answers.id match {
+                case RepresenteeCgtReference(cgtReference) => messages("representee.cyaLabel.cgtReference") + cgtReference.value
+                case _ => ""
```

Comment:
do we not want to display the other id types as well?

Reply:
Yes its confirmed by Shraddha Gurung and Jean Sims to avoid security issues.